libs/island-ui/contentful/src/lib/richTextRendering.tsx

```diff
@@ -35,24 +34,37 @@ import { TeamList, TeamListProps } from './TeamList/TeamList'
 import { ContactUs, ContactUsProps } from './ContactUs/ContactUs'
 import { Location, LocationProps } from './Location/Location'
 
-type HtmlSlice = { __typename: 'Html'; document: any }
-type FaqListSlice = { __typename: 'FaqList' } & FaqListProps
-type StatisticsSlice = { __typename: 'Statistics' } & StatisticsProps
-type ImageSlice = { __typename: 'Image' } & Omit<ImageProps, 'thumbnail'>
-type AssetSlice = { __typename: 'Asset' } & AssetLinkProps
-type ProcessEntrySlice = { __typename: 'ProcessEntry' } & ProcessEntryProps
-type EmbeddedVideoSlice = { __typename: 'EmbeddedVideo' } & EmbeddedVideoProps
-type TeamListSlice = { __typename: 'TeamList' } & TeamListProps
-type LocationSlice = { __typename: 'Location' } & LocationProps
-type ContactUsSlice = { __typename: 'ContactUs' } & Omit<
+type HtmlSlice = { __typename: 'Html'; id: string; document: Document }
```

Comment:
Cf this comment on the previous PR [here](https://github.com/island-is/island.is/pull/1083#discussion_r496581862). Did you check if it doesn't produce errors anymore?